test/test_app_groups.rb

```diff
@@ -37,6 +46,30 @@ def teardown
     # Do nothing
   end
 
+
+  # convenience function to create a reasonable group name
+  def create_group_name(root)
```

Comment:
those functions are also defined in the test_helper.rb file and utility_tasks.rb. Can we eliminate the duplicates?